SleepEngine/generator/Pipfile

```diff
@@ -0,0 +1,9 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+sleep-generator = {editable = true,path = "."}
```

Comment:
Please, add pipfiles and all python garbage in gitignore

Reply:
i think it's added in the your merge that is not yet in master. if so, i'll wait and get the changes from there. why do it twice

Reply:
Ok

Reply:
hey, there are pipfile in gitignore in your branch now. How are they here?)